modules/prebidServerBidAdapter/index.js

```diff
@@ -728,6 +729,8 @@ Object.assign(ORTB2.prototype, {
       if (floor) {
         imp.bidfloor = floor.floor;
         imp.bidfloorcur = floor.currency;
+        imp.ext.prebid.floors = { floorMin: imp.bidfloor };
```

Comment:
I'm not sure imp.ext.prebid always exists at this point; I would prefer `deepSetValue(imp, 'ext.prebid.floors.floorMin', imp.bidfloor)`. 

(I believe in some cases, like stored impressions where no params were set in `imp.ext.prebid.bidder`, this as it is would throw an error).

Reply:
actually I wonder - if the pub already set `ortb2Imp.ext.prebid.floors.floorMin`, should we keep it or override it here? I think keeping it makes more sense but then that should also become the value you check for against the request-wide `floorMin`.

Reply:
hmm, not sure.  so the question is should `imp.ext.prebid.floors.floorMin` be set to `ortb2Imp.ext.prebid.floors.floorMin` (if exists) or `imp.bidfloor` as a fallback?

inside of the description for https://github.com/prebid/Prebid.js/issues/8749, it says `if imp[n].bidfloor exists, copy it to imp[n].ext.prebid.floors.floorMin`. also states that `imp[n].bidfloor` is the default floor and that imp-level minFloor is not yet supported server-side.  does that mean (for now at least), that the logic should be `if imp[n].bidfloor exists, copy it to imp[n].ext.prebid.floors.floorMin`?

maybe @bretg could provide some more clarity around this?

Reply:
In my opinion if the pub has provided `ortb2Imp.ext.prebid.floors.floorMin` we should keep it and use that as the floorMin. But if the request-wide floor min is higher, than it should set it to the higher one.

The imp level floor mins should not overwrite the pub provided one if they are lower. Maybe I am misinterpreting though.

Reply:
@dgirardi @robertrmartinez could you review when you have time?  Spoke with @bretg on the issue page for this ticket (`https://github.com/prebid/Prebid.js/issues/8749`) and updated my PR just now.